backend/src/main/java/de/wevsvirushackathon/coronareport/report/HDReportController.java

```diff
@@ -10,7 +10,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/report")
+@RequestMapping("/hd/report")
```

Comment:
How about moving this also structurally into a sub-package? e.g. [...].hd.report instead of [...].report. This might include renaming [...].healthdepartment to [...].hd